webapp/src/dogma/common/components/Filter.tsx

```diff
@@ -0,0 +1,27 @@
+import { Column } from '@tanstack/react-table';
+import { DebouncedInput } from '@/dogma/common/components/DebouncedInput';
```

Comment:
Question: Is `@/` mandatory for the import path?

Reply:
Added them since the unit tests could not locate the components being tested. 
It's not needed anymore after removing the file mapper rule from jest config. Removed.